projects/ids-enterprise-ng/src/lib/lookup/soho-lookup.component.ts

```diff
@@ -483,6 +495,34 @@ export class SohoLookupComponent extends BaseControlValueAccessor<any> implement
       this.settings.options = Object.assign(datagridConfig, this.options);
       this.jQueryElement.lookup(this.settings);
 
+      // setup autocomplete
+      if (this.isAutoComplete) {
+        const acTemplate =`<script id="autocomplete-template-lookup" type="text/html">
```

Comment:
This template is hardcoded perhaps instead the template can be in the page so its adjustable. Then just pass the template id in the autoComplete settings. By default this should use the single / normal template https://main-enterprise.demo.design.infor.com/components/autocomplete/example-index.html